### DISCUSSION TITLE
Is it possible animate only outer div not the inner div?

### DISCUSSION THREAD

**Discussion by saifulshihab** [tailwindlabs/tailwindcss#4910] [Help] [answered]
I want to animate-pulse only outer div. 

```javascript
<div className="__outer__div animate-pulse">
   <div className="__inner_div ">
         ....
   </div>
</div>
```

Is there any way to not animate all staff inside the outer div???

**Comment by Martleby** [comment, marked as answer]
I've done something similar to this - have a look at this play:

https://play.tailwindcss.com/zhv66K7KIQ

The trick is to make the pinging element a child of the outer div, set it to absolute. The outer div should then set to relative. For any content you want inside the outer div, create another div and set this to relative with a positive z-index to place it over the top of the pinging.

Cheers,
Martin.

**Reply by saifulshihab** [reply]
Thanks. It's working. Actually, I also tried this way, but forgot to add `inset-0`.
